manifests/deploy/upstream/olm/metering.catalogsource.yaml

```diff
@@ -1,10 +1,10 @@
-apiVersion: operators.coreos.com/v1
+apiVersion: operators.coreos.com/v1alpha1
 kind: CatalogSource
 metadata:
-  name: installed-upstream-metering-operators-metering
-  namespace: marketplace
+  name: installed-redhat-metering-operators-openshift-metering
+  namespace: openshift-marketplace
```

Comment:
Is this a valid namespace in a vanilla Kubernetes cluster?

Reply:
that was a copy error...will use the Makefile to update instead of manually changing these.  thx!